script/Ciel/Day 5/05_01_CIEL05_1DASH.txt

```diff
@@ -285,7 +285,7 @@ It's like a pathological obsession. And yet, the work is messy.
 [sha:65f4446adad824e2997e22a77eb40f27e3973cc6]{
 -- Page 41, Offset 10975.
 -- 鉄板の大きさは窓の大きさと微妙に一致していない為、わずかな隙間から陽差しが入っている。
-The size of the steel plate doesn't quite match the size of the window,
+The size of the steel plates don't quite match the size of the windows,
```

Comment:
"doesn't" was right because "size" is the match term, not "plates/windows"

Reply:
I was right